src/redux/reducer/jobReducer.tsx

```diff
@@ -59,13 +60,48 @@ export const paginationApi = (currentPage: number) => {
   };
 };
 
-export const searchJobApi = (searchValue: string) => {
+export const searchJobByTitleApi = (searchValue: string) => {
   return async (dispatch: DispatchType) => {
     dispatch(changeIsLoading(true));
-    const result = await http.get(`/jobs/?search=${searchValue}`);
+    const result = await http.get(`/jobs/?jobTitle=${searchValue}`);
     if (result.data.length === 0) {
-      alert("Sorry, we do not find any jobs in the database");
-      window.location.href = "http://localhost:3000";
+      Swal.fire({
```

Comment:
Please show the text when search result is empty.

Reply:
Fixed